global-api/routes/city_locode_endpoint.py

```diff
@@ -0,0 +1,135 @@
+from fastapi import FastAPI
+from sqlalchemy import create_engine, text 
+from sqlalchemy.orm import sessionmaker
+
+app = FastAPI()
+
+# Read the DATABASE_URL from the .env file
+DATABASE_URL = config("DATABASE_URL")
```

Comment:
If we are going to use `decouple`, you have to add the dependency.

Also, why are we dependent on this .env file thing? Can't we read once from that file, and then just use the env values?

Reply:
this was a suggestion from @lemilonkh